test/units/testsuite-58.sh

```diff
@@ -7,7 +7,7 @@ runas() {
     declare userid=$1
     shift
     # shellcheck disable=SC2016
-    su "$userid" -s /bin/sh -c 'XDG_RUNTIME_DIR=/run/user/$UID exec "$@"' -- sh "$@"
+    su "$userid" -p -s /bin/sh -c 'XDG_RUNTIME_DIR=/run/user/$UID exec "$@"' -- sh "$@"
```

Comment:
Instead, how about to explicitly add `/usr/sbin` (and possibly `/sbin`) to `$PATH`?

Reply:
That would work too but what's wrong with the use of `-p` ?